neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -15,24 +15,46 @@
 #
 
 import logging
+import uuid
+from typing import List
 
 import click
+import dateutil
 import six
 from bravado.exception import HTTPBadRequest, HTTPNotFound, HTTPUnprocessableEntity
 from mock import NonCallableMagicMock
 
-from neptune.api_exceptions import ExperimentLimitReached, \
-    ExperimentValidationError, ProjectNotFound
-from neptune.exceptions import STYLES
+from neptune.alpha import exceptions as alpha_exceptions
+from neptune.alpha.internal import operation as alpha_operation
+from neptune.alpha.internal.backends.api_model import AttributeType as AlphaAttributeType
+from neptune.alpha.internal.backends.hosted_neptune_backend import HostedNeptuneBackend as AlphaHostedNeptuneBackend
+from neptune.alpha.internal.credentials import Credentials as AlphaCredentials
+from neptune.alpha.internal.utils import paths as alpha_path_utils
+from neptune.api_exceptions import (
+    AlphaOperationErrors,
+    ChannelNotFound,
+    ExperimentLimitReached,
+    ExperimentNotFound,
+    ExperimentValidationError,
+    ProjectNotFound,
+)
+from neptune.exceptions import STYLES, NeptuneException
+from neptune.experiments import Experiment
 from neptune.internal.backends.hosted_neptune_backend import HostedNeptuneBackend
+from neptune.internal.utils.alpha_integration import MONITORING_ATTRIBUTE_SPACE, PARAMETERS_ATTRIBUTE_SPACE
 from neptune.internal.utils.http import extract_response_field
+from neptune.model import AlphaChannelWithLastValue
 from neptune.projects import Project
 from neptune.utils import with_api_exceptions_handler
 
 _logger = logging.getLogger(__name__)
 
 
 class AlphaIntegrationBackend(HostedNeptuneBackend):
```

Comment:
Why is this inheritance necessary?

Reply:
That's something I've discussed earlier - it probably will be abandoned, but it's useful now:
https://github.com/neptune-ai/neptune-client/pull/364#discussion_r558226382

Reply:
Ok.